packages/builder/builder-webpack-provider/src/webpackPlugins/ProgressPlugin/helpers/bar.ts

```diff
@@ -76,14 +72,7 @@ export const renderBar = (option: Partial<Props>) => {
     chalk,
     messageColor,
   )(cliTruncate(message, messageWidth, { position: 'start' }));
-  const frameStr = fc(frames[current % frames.length]);
-  if (total <= 0) {
-    // 无法获取百分比时用动画表示进度
-    if (columns >= MIDDLE_WIDTH) {
-      return [idStr, frameStr, space, msgStr].join('');
-    }
-    return [idStr, frameStr].join('');
-  }
+
```

Comment:
I think `total` will not be minus yet, so the `total <= 0` condition is removed.